src/compressor/compressor12.pil.ejs

```diff
@@ -341,6 +344,11 @@ namespace Compressor(%N);
     g012*TREESELECTOR4 = 0;
     g345*TREESELECTOR4 = 0;
 
+    pol checkBinaryKey1 = a[6]'*(1 - a[6]');
+    pol checkBinaryKey2 = a[7]'*(1 - a[7]');
+    TREESELECTOR4 * checkBinaryKey1 = 0;
+    TREESELECTOR4 * checkBinaryKey2 = 0;
```

Comment:
The binary checks for the tree selector is not necessary if you would like to exclude.